test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -1,9 +1,23 @@
-import { createHash, randomBytes } from "crypto"
+import { Wallets } from "@app"
+import { intraledgerPaymentSendUsername } from "@app/wallets"
```

Comment:
use method from `Wallets`

Reply:
do we want to use the line 1 format here or line 2?